client/src/templates/Challenges/components/interactive-editor.tsx

```diff
@@ -43,8 +49,9 @@ const InteractiveEditor = ({ files }: Props) => {
   function got(ext: string) {
     return files.some(f => f.ext === ext);
   }
-
   const showConsole = got('js') || got('ts');
+  const isHtmlOnly = got('html') && !showConsole;
```

Comment:
This is confusing. There could be python, css, jsx and so on and this would still be `true`.